src/utils/getPrepearedTodos.ts

```diff
@@ -0,0 +1,23 @@
+import { FilterOptions } from '../types/FilterOptions';
+import { Todo } from '../types/Todo';
+
+export function getPrepearedTodos(todos: Todo[], filter: FilterOptions) {
+  let preparedTodos: Todo[] = [...todos];
+
+  if (filter !== FilterOptions.ALL) {
+    switch (filter) {
+      case FilterOptions.ACTIVE:
+        preparedTodos = preparedTodos.filter(todo => !todo.completed);
+        break;
+
+      case FilterOptions.COMPLETED:
+        preparedTodos = preparedTodos.filter(todo => todo.completed);
+        break;
+
+      default:
+        break;
+    }
+  }
```

Comment:
```suggestion

  if (filter === FilterOptions.ALL) {
    return preparedTodos;
  }
  
  switch (filter) {
    case FilterOptions.ACTIVE:
      return preparedTodos.filter(todo => !todo.completed);
      
    case FilterOptions.COMPLETED:
      return preparedTodos.filter(todo => todo.completed);

    default:
      break;
  }
```
it will be more readable